astropy/coordinates/sky_coordinate.py

```diff
@@ -41,6 +42,24 @@ def FRAME_ATTR_NAMES_SET():
     return out
 
 
+class SkyCoordColumnInfo(ColumnInfo):
+    def __init__(self, *args, **kwargs):
+        super(SkyCoordColumnInfo, self).__init__(*args, **kwargs)
+        self._format_cache = {}
+        if self._parent_col is not None:
+            data = self._parent_col().data
+            self.unit = ','.join(str(getattr(data, comp).unit) for comp in data.components)
+
+    def default_format_func(self, format, val):
```

Comment:
p.s. Is `.value` is necessarily the best option? Since the components could be `Angle`, one could also get hhmmss etc...  But that probably is best left for later.